### PR TITLE
update json example for TAG

### PR DESCRIPTION
**Pull request by emmanuelkeller**

### CODE REVIEW

File: docs/Indexing_JSON.md [hunk 1]
```diff
@@ -53,7 +50,7 @@ In our example we are going to use the following JSON document:
 
 We can use `JSON.SET` to store in our database:
 
-    JSON.SET myDoc $ '{"user":{"name":"John Smith","tag":["foo","bar"],"hp":1000, "dmg":150}}'
+    JSON.SET myDoc $ '{"user":{"name":"John Smith","tag":"foo.bar","hp":1000, "dmg":150}}'
```
Comment (ashtul) [inline docs/Indexing_JSON.md:53]: Should it be a `,`?

Comment (oshadmi) [reply docs/Indexing_JSON.md:53]: In my opinion, we better have corresponding tests for our documentation samples so we don't break them, or if we do, we don't have out-of-sync documentation👼🏼

File: docs/Indexing_JSON.md [hunk 2]
```diff
@@ -134,7 +131,7 @@ The LOAD parameters accepts JSON Path expressions. Any value (even not indexed)
 In this example we are loading two numeric values from the JSON document applying a simple operation.
 
 ```
-FT.AGGREGATE myIdx '*' LOAD 6 $.user.hp AS hp $.user.dmg AS dmg APPLY '@hp-@dmg' AS points
+FT.AGGREGATE userIdx '*' LOAD 6 $.user.hp AS hp $.user.dmg AS dmg APPLY '@hp-@dmg' AS points
```
Comment (oshadmi) [inline docs/Indexing_JSON.md:134]: Written above:

> Any value (even not indexed) can be used in the pipeline.

Is it a "value" (which is suitable for Hash) or actually a "property" (relevant for JSON)?

File: docs/Indexing_JSON.md [hunk 3]
```diff
@@ -66,11 +63,11 @@ We can search any attribute mentioned in the SCHEMA.
 Following our example, let's find our user called `John`:
 
 ```
-FT.SEARCH myIdx '@name:(John)'
+FT.SEARCH userIdx '@name:(John)'
 1) (integer) 1
 2) "myDoc"
 3) 1) "$"
-   2) "{\"user\":{\"name\":\"John Smith\",\"tag\":[\"foo\",\"bar\"],\"hp\":1000,\"dmg\":150}}"
+   2) "{\"user\":{\"name\":\"John Smith\",\"tag\":\"foo.bar\",\"hp\":1000,\"dmg\":150}}"
```
Comment (oshadmi) [inline docs/Indexing_JSON.md:70]:
```suggestion
   2) "{\"user\":{\"name\":\"John Smith\",\"tag\":\"foo,bar\",\"hp\":1000,\"dmg\":150}}"
```

File: docs/Indexing_JSON.md (hunk 2 above)
Comment (oshadmi) [reply docs/Indexing_JSON.md:134]: For later, need to decide what would be the best term for "value"

**Review by oshadmi — APPROVED** [review]
👍🏼

**Pull request merged**